src/storage/PouchDBStorage.ts

```diff
@@ -54,6 +54,24 @@ export class PouchDBStorage {
     this.sync.on("error", this.onSyncError.bind(this));
   }
 
+  // takes DB URL as a param
+  connectRemoteDBCustom(dbURL: string, username: string, password: string): void {
```

Comment:
Would you mind combining this with `connectRemoteDB()` using an optional `options` param (see below)?

Reply:
Sure. Go ahead